react-charts/area-chart/step3.md

```diff
@@ -1,53 +1,61 @@
-import './app.css';
-import {
-  Chart,
-  ChartAxis,
-  ChartArea,
-  ChartGroup,
-  ChartThemeColor,
-  ChartVoronoiContainer
-} from '@patternfly/react-charts';
-
-function App() {
-  return (
-    &lt;div className=&quot;chart-container&quot;&gt;
-      &lt;Chart
-        height={250}
-        width={600}
-      &gt;
-        &lt;ChartArea
-          data={[
-            { name: &#39;Cats&#39;, x: &#39;2015&#39;, y: 3 }, 
-            { name: &#39;Cats&#39;, x: &#39;2016&#39;, y: 4 }, 
-            { name: &#39;Cats&#39;, x: &#39;2017&#39;, y: 8 }, 
-            { name: &#39;Cats&#39;, x: &#39;2018&#39;, y: 6 }
-          ]} 
-          interpolation="basis"
-        /&gt;
-      &lt;/Chart&gt;
-    &lt;/div&gt;
-  );
-}
-
-export default App;
-</pre>
-
-- The `interpolation` property adds interpolating curves
-
-3) When the server reloads, you should see something like this:
-<img src="area-chart/assets/simple.png" alt="Simple chart" style="box-shadow: rgba(3, 3, 3, 0.2) 0px 1.25px 2.5px 0px;" />
-
-We'll continue by adding multiple datasets to the chart in the next step.
+Now you will add multiple datasets to the area chart.
+
+## Task
+
+1) Make sure the <strong>App.js file is still open.</strong>
+
+2) <strong>Locate the code that,</strong> looks like the following:
+
+<pre class="file">
+&lt;ChartArea
+  data={[
+    { name: &#39;Cats&#39;, x: &#39;2015&#39;, y: 3 }, 
+    { name: &#39;Cats&#39;, x: &#39;2016&#39;, y: 4 }, 
+    { name: &#39;Cats&#39;, x: &#39;2017&#39;, y: 8 }, 
+    { name: &#39;Cats&#39;, x: &#39;2018&#39;, y: 6 }
+  ]} 
+/&gt;
+</pre>
+
+3)<strong> Replace that section</strong> with the following code:
+
+<pre class="file" data-target="clipboard">
+&lt;ChartGroup&gt;
+  &lt;ChartArea
+    data={[
+      { name: &#39;Cats&#39;, x: &#39;2015&#39;, y: 3 }, 
+      { name: &#39;Cats&#39;, x: &#39;2016&#39;, y: 4 }, 
+      { name: &#39;Cats&#39;, x: &#39;2017&#39;, y: 8 }, 
+      { name: &#39;Cats&#39;, x: &#39;2018&#39;, y: 6 }
+    ]}
+    interpolation="basis"
+  /&gt;
+  &lt;ChartArea 
+    data={[
+      { name: &#39;Dogs&#39;, x: &#39;2015&#39;, y: 2 }, 
+      { name: &#39;Dogs&#39;, x: &#39;2016&#39;, y: 3 }, 
+      { name: &#39;Dogs&#39;, x: &#39;2017&#39;, y: 4 }, 
+      { name: &#39;Dogs&#39;, x: &#39;2018&#39;, y: 5 },
+      { name: &#39;Dogs&#39;, x: &#39;2019&#39;, y: 6 }
+    ]}
+    interpolation="basis"
+  /&gt;
+  &lt;ChartArea 
+    data={[
+      { name: &#39;Birds&#39;, x: &#39;2015&#39;, y: 1 }, 
+      { name: &#39;Birds&#39;, x: &#39;2016&#39;, y: 2 }, 
+      { name: &#39;Birds&#39;, x: &#39;2017&#39;, y: 3 }, 
+      { name: &#39;Birds&#39;, x: &#39;2018&#39;, y: 2 },
+      { name: &#39;Birds&#39;, x: &#39;2019&#39;, y: 4 }
+    ]}
+    interpolation="basis"
+  /&gt;
+&lt;/ChartGroup&gt;
+</pre>
+
+<strong>Note:</strong> The `ChartGroup` is the React Component used to apply properties to multiple `ChartArea` components.
+
+Once the preview reloads - it should look like this:

```

Comment:
Let's replace the em dash with a comma so that it reads: Once the preview loads, it should look like this: